cli/tests/integrations/test_experimental.py

```diff
@@ -301,37 +300,37 @@ def _service_start_cmd(package_name,
                        package_version=None,
                        options=None,
                        json=False):
-    return (command_base
-            + (['service', 'start'])
-            + (['--json'] if json else [])
-            + ([package_name])
-            + (['--package-version', package_version]
-               if package_version else [])
-            + (['--options', options] if options else []))
+    return (command_base +
+            (['service', 'start']) +
+            (['--json'] if json else []) +
+            ([package_name]) +
+            (['--package-version', package_version]
+             if package_version else []) +
+            (['--options', options] if options else []))
 
 
 def _package_add_cmd(dcos_package=None,
                      package_name=None,
                      package_version=None,
                      json=False):
-    return (command_base
-            + (['package', 'add'])
-            + (['--json'] if json else [])
-            + (['--dcos-package', dcos_package] if dcos_package else [])
-            + (['--package-name', package_name] if package_name else [])
-            + (['--package-version', package_version]
-               if package_version else []))
+    return (command_base +
+            (['package', 'add']) +
+            (['--json'] if json else []) +
+            (['--dcos-package', dcos_package] if dcos_package else []) +
+            (['--package-name', package_name] if package_name else []) +
+            (['--package-version', package_version]
+             if package_version else []))
```

Comment:
Formatting change. In PEP8 multi-line expression should have each line end in a binary operation.